tests/playwright/Report.md

```diff
@@ -0,0 +1,124 @@
+# Evaluation of Microsoft Playwright
+
+## Introduction
+**Automation testing**
+Automation testing is a way to automate one's browser by simulating user actions like clicks. As Software or applications become more complex as they are built on layers and entire networks of sub-systems, including UI and API layers, external databases, networks, and even third-party integrations, there is always a need for thorough testing to be done, this ranges from unit testing to end to end testing of the application. Browser automation frameworks to perform end-to-end testing helps to ascertain the stability and the health of the application which in turn leads to the success of the application when on production. The general stability of the application doesn't entirely depend on end-to-end testing but it surely helps detects bugs, among other benefits, thus increasing application productivity. Below are some of the popular test automation frameworks;
+- Selenium
+- Puppeteer
+- Cypress
+- Playwright(maybe the new kid on the block but it's gaining quite a trajectory among Software developers and testers)
+
+### Project Context
+The Playwright evaluation is a result of a need for potential replacement of [WebdriverIO](https://webdriver.io/) with one of the best non-selenium modern automation testing framwork. Wikimedia supports a large number of repositories as well as extensions. In order to ensure good code practices across all these repositories, an extensive amount of testing is performed and one of the tests performed is an end to end test. WebdriverIO is the current browser automation framework being used for implementing the end-to-end tests. However, with the recent increase in awareness about end-to-end testing, a number of equally competitive non-selenium solutions have been introduced and one of them is [Microsoft Playwright](https://github.com/microsoft/playwright).
```

Comment:
> repositories as well as extensions

I'm not sure why you are making a difference here. Most mediawiki extensions are git repositories. I would pick either repositories or extensions, no need to mention both.

Reply:
Noted. Thanks